libs/client/shared/ui/view-header/src/lib/view-header/view-header.component.ts

```diff
@@ -7,16 +7,25 @@ import {
 } from '@angular/core';
 import { JsonPipe } from '@angular/common';
 import { UiViewHeader } from './view-header.interface';
-
+import { TuiButtonModule, TuiSvgModule } from '@taiga-ui/core';
 @Component({
   selector: 'ddd-view-header',
   standalone: true,
-  imports: [JsonPipe],
+  imports: [JsonPipe, TuiButtonModule, TuiSvgModule ],
   templateUrl: './view-header.component.html',
   styleUrls: ['./view-header.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class ViewHeaderComponent {
   @Input() header: UiViewHeader | null = null;
-  @Output() backButtonClicked = new EventEmitter<void>();
+  @Output() prevButtonClicked = new EventEmitter<void>();
+  @Output() nextButtonClicked = new EventEmitter<void>();
```

Comment:
we don't need this button nextButtonClicked. In the context of a view, e.g. Parts List. because you don't know what the next button means. Where then do you have to move the user when he clicks the button? For part details view? If so, how do you define which part it is?